src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,58 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int OPERATION_TYPE = 0;
+    public static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder fromFileContent = new StringBuilder();
+        StringBuilder toFileContent = new StringBuilder();
```

Comment:
Your public method would be something like
```suggestion
    public void getStatistic(String fromFileName, String toFileName) {
        String[] fileContent = readFileContent(fromFileName);
        String Report = makeReport(fileContent);
        writeToFile(report);
    }
```
and logic is split between these private methods